packages/widgets-internal/farm/components/FarmApyButton/index.tsx

```diff
@@ -38,12 +38,12 @@ export const FarmApyButton: React.FC<React.PropsWithChildren<FarmApyButtonProps>
         style={{ textDecoration: strikethrough ? "line-through" : "initial" }}
         onClick={handleClick}
       >
-        {children}
```

Comment:
will affect the other page, such as aptos/farms?

Reply:
yes, it will. just UI changes